src/uct/tcp/sockcm/sockcm_iface.c

```diff
@@ -92,31 +122,143 @@ static uct_iface_ops_t uct_sockcm_iface_ops = {
     .iface_get_address        = uct_sockcm_iface_get_address
 };
 
-void uct_sockcm_iface_client_start_next_ep(uct_sockcm_iface_t *iface)
+static void uct_sockcm_iface_event_handler(int fd, void *arg)
 {
-    ucs_trace("sockcm_iface_client_start_next_ep not implemented");
+    ucs_debug("not implemented yet");
 }
 
 static UCS_CLASS_INIT_FUNC(uct_sockcm_iface_t, uct_md_h md, uct_worker_h worker,
                            const uct_iface_params_t *params,
                            const uct_iface_config_t *tl_config)
 {
+    uct_sockcm_iface_config_t *config = ucs_derived_of(tl_config, uct_sockcm_iface_config_t);
+    char ip_port_str[UCS_SOCKADDR_STRING_LEN];
+    ucs_status_t status;
+    int ret = 0;
```

Comment:
plz, move it up to have all initialized vars in the beginning

Reply:
I've removed this var because it's not really used other than to check if return is non-zero